### PR TITLE
Update create_graph.py

### PR DESCRIPTION
**Pull request by bsantan**
Convert edgeless column names to lower case

### CODE REVIEW

File: create_graph.py [hunk 1]
```diff
@@ -16,6 +16,9 @@ def process_pkl_files(triples_file,labels_file):
 
 #Creates igraph object and a list of nodes
 def create_igraph_graph(edgelist_df,labels):
+    
+    #To ensure using column names is case insensitive
+    edgelist_df.columns = edgelist_df.columns.str.lower()
```
Comment (lagillenwater) [inline create_graph.py:21]: will this cause an error in the next line which reorders the df?

Comment (bsantan) [reply create_graph.py:21]: It didn't for me, it's just changing the columns names from Subject --> subject for example. Unless I am missing something?

Comment (lagillenwater) [reply create_graph.py:21]: I got this error while running it: "KeyError: "None of [Index(['Subject', 'Object', 'Predicate'], dtype='object')] are in the [columns]"

Comment (bsantan) [reply create_graph.py:21]: I can't reproduce this. I wonder if it was how I was passing the input files but I might just make the changes for the other issues I have and see if this is resolved

Comment (lagillenwater) [reply create_graph.py:21]: ok. sounds good.

**Review by Nourah-Salem — APPROVED** [review]

**Review by Nourah-Salem — APPROVED** [review]

**Comment by bsantan** [comment]
Input labels and triples file column names to lower is handled in assign_nodes branch

**Pull request closed without merging**